src/apps/service/apps/de/listings.clj

```diff
@@ -687,7 +687,7 @@
 (defn- format-task
   [user {:keys [tool_id] :as task}]
   (-> task
-      (assoc :tool (when tool_id (tools/get-tool user tool_id)))
+      (assoc :tool (when tool_id (tools/get-tool user tool_id true)))
```

Comment:
I _think_ this is all that's needed to get the default values passed down to app-exposer/jex-adapter, but it feels a bit too easy so I feel like I must have missed something, haha.

Reply:
I feel that way a lot when I'm doing Clojure programming. 😆